### PR TITLE
Node16

### PR DESCRIPTION
**Pull request by ags1773**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,28 +181,38 @@ isomorphicRoutes(app, {
...
-
-         return self.registration.showNotification(notificationTitle,
-           notificationOptions);
-       }
-       messaging.setBackgroundMessageHandler(messageHandler);
+   ```js
+    importScripts("https://www.gstatic.com/firebasejs/9.6.10/firebase-app-compat.js");
+    importScripts("https://www.gstatic.com/firebasejs/9.6.10/firebase-messaging-compat.js");
+
+    var firebaseConfig =  <%- JSON.stringify(firebaseConfig ) %>
```
Comment (Jeevan-Kishore) [inline README.md:188]:
```suggestion
    const firebaseConfig =  <%- JSON.stringify(firebaseConfig ) %>
```

File: README.md [hunk 2]
```diff
@@ -174,28 +181,38 @@ isomorphicRoutes(app, {
...
+    firebase.initializeApp(firebaseConfig);
+    self.addEventListener('notificationclick', function(event) {
+      var url = event.notification.data.url;
```
Comment (Jeevan-Kishore) [inline README.md:191]:
```suggestion
      const url = get(event, ["notification", "data", "url"]);
```

File: README.md [hunk 3]
```diff
@@ -174,28 +181,38 @@ isomorphicRoutes(app, {
...
-           notificationOptions);
-       }
-       messaging.setBackgroundMessageHandler(messageHandler);
+   ```js
+    importScripts("https://www.gstatic.com/firebasejs/9.6.10/firebase-app-compat.js");
+    importScripts("https://www.gstatic.com/firebasejs/9.6.10/firebase-messaging-compat.js");
+
+    var firebaseConfig =  <%- JSON.stringify(firebaseConfig ) %>
+    firebase.initializeApp(firebaseConfig);
+    self.addEventListener('notificationclick', function(event) {
```
Comment (Jeevan-Kishore) [inline README.md:190]:
```suggestion
    self.addEventListener('notificationclick', (event) => {
```

File: README.md [hunk 4]
```diff
@@ -174,28 +181,38 @@ isomorphicRoutes(app, {
...
+        clients.openWindow(url);
+      }
+      clients.openWindow('<%= sketchesHost %>');
+    }, false);
+
+    const messaging = firebase.messaging();
+
+    messaging.onBackgroundMessage(function(payload) {
+      const data = payload["data"];
+      const notificationTitle = data.title;
```
Comment (Jeevan-Kishore) [inline README.md:202]:
```suggestion
      const notificationTitle = get(data, ["title"]);
```

**Review by Jeevan-Kishore — CHANGES_REQUESTED** [review]

**Review by hbindiya — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -174,28 +181,38 @@ isomorphicRoutes(app, {
...
-       }
-       messaging.setBackgroundMessageHandler(messageHandler);
+   ```js
+    importScripts("https://www.gstatic.com/firebasejs/9.6.10/firebase-app-compat.js");
+    importScripts("https://www.gstatic.com/firebasejs/9.6.10/firebase-messaging-compat.js");
+
+    const firebaseConfig =  <%- JSON.stringify(firebaseConfig ) %>
+    firebase.initializeApp(firebaseConfig);
+    self.addEventListener('notificationclick', (event) => {
+      const url = get(event, ["notification", "data", "url"]);
```
Comment (ags1773) [inline README.md:191]: @Jeevan-Kishore lodash get won't be defined here right?

Comment (Jeevan-Kishore) [reply README.md:191]: Lets use `||` and error safe it

**Pull request merged**